server/src/test/omtLinkProvider.test.ts

```diff
@@ -0,0 +1,132 @@
+import { expect, use } from 'chai';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import { stub } from 'sinon';
+import { DocumentLink, Position, Range } from 'vscode-languageserver';
+import { TextDocument } from 'vscode-languageserver-textdocument';
+import OMTLinkProvider from '../omtLinkProvider';
+import { WorkspaceLookup } from '../workspaceLookup';
+import * as sinonChai from 'sinon-chai';
+use(sinonChai);
+
+type Case = {
+    i: number,
+    line: number,
+    start: number,
+    end: number,
+    should: string,
+    target: string | undefined,
+    module?: string,
+};
+
+describe('OMTLinkProvider', () => {
+    let linkProvider: OMTLinkProvider;
+    let lookupStub: WorkspaceLookup;
+
+    beforeEach(() => {
+        lookupStub = <WorkspaceLookup>{
+            // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Does eslint object not using `_module`? Because tslint always ignored unused parameters which start with an underscore

Reply:
nope, sadly it doesn't. it will be shown darker by the IDE indicating that isn't used, but eslint still gives a warning